src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -117,6 +109,19 @@ class EmojiPickerMenu extends Component {
         this.setFirstNonHeaderIndex(this.emojis);
     }
 
+    // eslint-disable-next-line rulesdir/prefer-early-return
+    componentDidUpdate(prevProps) {
+        if (prevProps.frequentlyUsedEmojis !== this.props.frequentlyUsedEmojis) {
+            const {filteredEmojis, headerRowIndices} = this.getInitialFilteredEmojisAndHeaderRowIndices();
+            this.emojis = filteredEmojis;
+            this.headerRowIndices = headerRowIndices;
+            this.setState({
+                filteredEmojis: this.emojis,
+                headerIndices: this.headerRowIndices,
+            });
+        }
+    }
```

Comment:
Can you flip this condition and return early, rather than disabling eslint rule?